soda_machine.rb

```diff
@@ -7,12 +7,19 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.count
   end
 
   def find_soda(soda_brand)
+    @sodas
+      # @sodas.index(soda_brand)
+      # @sodas.select {soda_brand}
   end
```

Comment:
Hi Sarah,

Yes I think I need that. Should I book a time on your schedule or I should
just go to you when you don't look busy.

Sincerely,

Hank

Sarah Spikes <notifications@github.com> 於 2016年5月10日 星期二寫道：

> In soda_machine.rb
> <https://github.com/sf-pocket-gophers-2016/soda-machine-challenge/pull/7#discussion_r62735898>
> :
>
> >    end
> >
> >    def find_soda(soda_brand)
> > +    @sodas
> > +      # @sodas.index(soda_brand)
> > +      # @sodas.select {soda_brand}
> >    end
>
> Let's find some time to sit down and chat about this challenge.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sf-pocket-gophers-2016/soda-machine-challenge/pull/7/files/48d311666472aec70f4bc177f0b30ce930745e22#r62735898>
>